spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/ConformanceExecution.scala

```diff
@@ -43,37 +46,37 @@ import za.co.absa.enceladus.utils.schema.SchemaUtils
 import scala.util.control.NonFatal
 import scala.util.{Failure, Success, Try}
 
-trait ConformanceExecution extends CommonJobExecution {
+trait ConformanceExecution extends CommonJobExecution with S3DefaultCredentialsProvider {
   private val conformanceReader = new ConformancePropertiesProvider
   private val sourceId = SourcePhase.Conformance
 
   protected def prepareConformance[T](preparationResult: PreparationResult)
                                      (implicit dao: MenasDAO,
                                       cmd: ConformanceConfigParser[T],
-                                      fsUtils: FileSystemVersionUtils,
+                                      fsUtils: DistributedFsUtils,
                                       spark: SparkSession): Unit = {
-    //val stdDirSize = fsUtils.getDirectorySize(preparationResult.pathCfg.standardizationPath)
-    //preparationResult.performance.startMeasurement(stdDirSize) // TODO fix for s3 [ref issue #1416]
+
+    val stdDirSize = fsUtils.getDirectorySize(preparationResult.pathCfg.standardizationPath)
+    preparationResult.performance.startMeasurement(stdDirSize)
 
     log.info(s"standardization path: ${preparationResult.pathCfg.standardizationPath}")
     log.info(s"publish path: ${preparationResult.pathCfg.publishPath}")
 
-    // Enable Control Framework
```

Comment:
So this is not needed anymore?

Reply:
I moved the comment closer to where it makes more sense.

Reply:
Aa, so this way that import is not needed anymore